From e2435e253819e1c6868238c0714c4a1d6c7667e3 Mon Sep 17 00:00:00 2001
From: Jiangli Yuan <jlyuan@motorola.com>
Date: Fri, 22 Jan 2016 14:44:02 -0800
Subject: [PATCH 639/959] IKSWM-17762  8952/defconfig: Enable eMMC cmd queue

 Enable CONFIG_MMC_CQ_HCI

Signed-off-by: Jiangli Yuan <jlyuan@motorola.com>

Change-Id: I1f0485451760d9f2a3bea7835bd9ee552373328b
Reviewed-on: http://gerrit.mot.com/815593
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Sheng-Zhe Zhao <a18689@motorola.com>
Reviewed-by: Zhi-Ming Yuan <a14194@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 arch/arm/configs/ext_config/moto-msm8916.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/ext_config/moto-msm8916.config b/arch/arm/configs/ext_config/moto-msm8916.config
index f200be49cfa..0dba08f337d 100644
--- a/arch/arm/configs/ext_config/moto-msm8916.config
+++ b/arch/arm/configs/ext_config/moto-msm8916.config
@@ -60,3 +60,4 @@ CONFIG_SUSPEND_DEBUG=y
 CONFIG_ALSA_TO_H2W=y
 CONFIG_ESD_FS=y
 CONFIG_SEC_DRIVER=y
+CONFIG_MMC_CQ_HCI=y
-- 
2.11.0

